lib/kramdown-rfc/command.rb

```diff
@@ -490,7 +490,7 @@ def expand_tabs(s, tab_stops = 8)
 $options ||= OpenStruct.new
 op = OptionParser.new do |opts|
   opts.banner = <<BANNER
-Usage: kramdown-rfc2629 [options] file.md|file.mkd > file.xml
+Usage: kramdown-rfc2629 [options] [MARKDOWNFILE] > file.xml
```

Comment:
I like the square brackets.
I don't like the jarring uppercase thing.
I have seen this kind of usage information (using | for or) a lot; too lazy now to look up examples.
But I think I only want to keep the brackets.